src/pages/document/invite-new-user-modal.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react"
+import { Platform } from "react-native"
+import { DialogModal } from "../../widgets/modal"
+import { useTranslation } from "react-i18next"
+import { Group, Column, Row } from "../../layout"
+import { Form, LabelText, TextField, CheckBox } from "../../forms"
+import { Text } from "../../text"
+import Button from "../../widgets/button"
+import { SearchAndTag } from "../../forms/search-and-tag"
+import Tooltip, { TooltipIcon } from "../../widgets/tooltip"
+
+export default function InviteNewUserModal(props) {
+	const [t] = useTranslation()
+
+	const { firstName, lastName, artistName, email, groups } = props
+
+	const [search, setSearch] = useState("")
+	const [selected, setSelected] = useState([])
+
+	return (
+		<DialogModal
+			visible={props.visible}
+			onRequestClose={props.onRequestClose}
+			title={t("newUserInvite:title")}
+			buttons={
+				<>
+					<Button
+						text={t("general:buttons.cancel")}
+						tertiary
+						onClick={props.onRequestClose}
+					/>
+					<Button text={t("general:buttons.save")} onClick={props.onSubmit} />
+				</>
+			}
+		>
+			<Group
+				of="group"
+				style={Platform.OS === "web" && { minWidth: 560, alignSelf: "center" }}
+			>
+				<Form
+					values={{ firstName } + { lastName }}
+					onSubmit={(values) =>
+						alert("Submit " + JSON.stringify(values, null, 4))
+					}
+				>
+					<Column of="group">
+						<Row of="component">
+							<TextField
+								value={firstName}
+								label={t("forms:labels.legalFirstName")}
+							/>
+
+							<TextField
+								value={lastName}
+								label={t("forms:labels.legalLastName")}
+							/>
+						</Row>
+
+						<TextField
+							value={artistName}
+							label={t("forms:labels.artistName")}
+							abel_hint={t("forms:labels.optional")}
+							undertext={t("forms:undertexts.artistName")}
+						/>
+
+						<TextField value={email} label={t("forms:labels.email")} />
+
+						<SearchAndTag
+							label={t("forms:labels.groups")}
+							searchResults={props.searchResults}
+							searchInput={search}
+							onSearchChange={setSearch}
+							selectedItems={selected}
+							onSelect={(selection) => setSelected([...selected, selection])}
+							onUnselect={(selection) =>
+								setSelected(selected.filter((i) => i !== selection))
+							}
+							placeholder={t("forms:placeholders.groupSearch")}
+						/>
+
+						<LabelText>{t("newUserInvite:checkbox")} </LabelText>
```

Comment:
Mets tout le reste dans un `<Column of="component">` pour avoir le bon espacement

Avant: 
![image](https://user-images.githubusercontent.com/5481612/86695894-f60c5300-bfda-11ea-9a73-59e639e5dad0.png)

Après: 
![image](https://user-images.githubusercontent.com/5481612/86696014-0fad9a80-bfdb-11ea-9510-c2c611765bbe.png)